pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/mercury/MercuryOrderDao.java

```diff
@@ -207,9 +219,8 @@ public boolean orderNumberExists(String orderNumber) {
                     } else {
                         dbVals.resultValue = false;
                     }
-
-                } catch (Exception e) {
-                    throw new RuntimeException("Error getting values from db", e);
+                } catch (SQLException exception) {
```

Comment:
This is unnecessary since you are catching the same exception right below. Also, this method can be simplified by directly returning the results of the transaction, and not using SimpleResult, which does not add value.